File: java/AreaOfCircle.java
import java.util.Scanner;
class AreaOfCircle
{
	public static void main(String args[])
	{
		Scanner kb=new Scanner(System.in);
		System.out.print("Enter radius of circle : ");
		int r=kb.nextInt();

		float area;

		area=3.14f*r*r;
		System.out.println("Area is "+area);
	}
}

File: c/area-of-circle.c
#include<stdio.h>
int main()
{
	int radius;
	float area;
	
	printf("Enter radius of the circle : ");
	scanf("%d",&radius);
	
	area=3.14*radius*radius;
	
	printf("Area of circle with radius %d is %f",radius,area);
	return 0;
}
